src/main/java/com/example/MyActions.java

```diff
@@ -0,0 +1,11 @@
+package com.example;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class MyActions {
```

Comment:
Consider renaming this to something more specific to Spring Boot, like ActionsSpringBootApp or ActionsSpringBootApplication.